### PR TITLE
sentencepiece: add rake task to build deb package

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: sentencepiece/Rakefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+require "rake"
+require "octokit"
+
+package_task = ENV["RED_DATA_TOOLS_REPOSITORY"] + "/vendor/arrow/dev/tasks/linux-packages/package-task"
```
Comment (kou) [inline sentencepiece/Rakefile]: Could you refer the original Apache Arrow repository instead of Red Data Tools repository?

Comment (kenhys) [reply sentencepiece/Rakefile]: Thanks, I've fixed to use apache/arrow/dev...

File: sentencepiece/Rakefile [hunk 2]
```diff
@@ -0,0 +1,122 @@
+require "rake"
+require "octokit"
+require "open-uri"
+
+UBUNTU_CODE_NAMES = "xenial,bionic,disco"
+UBUNTU_VERSIONS = "16.04,18.04,19.04"
+PACKAGE = "sentencepiece"
+LAUNCHPAD_PPA ||= "groonga-ppa"
```
Comment (kou) [inline sentencepiece/Rakefile]: `||=` is meaningless here.

File: sentencepiece/Rakefile [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+package_task = ENV["APACHE_ARROW_REPOSITORY"] + "/dev/tasks/linux-packages/package-task"
+
+ENV["APT_TARGETS"] ||= "debian-buster"
+
+require "#{package_task}"
```
Comment (kou) [inline sentencepiece/Rakefile]: `"#{...}"` is meaningless here.

File: sentencepiece/Rakefile [hunk 4]
```diff
@@ -0,0 +1,122 @@
+require "rake"
```
Comment (kou) [inline sentencepiece/Rakefile]: We don't need this.

File: sentencepiece/Rakefile [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  private
+  def source_dir
+    File.join(__dir__, "sentencepiece-#{@version}")
+  end
+
+  def detect_version
+    @client = Octokit::Client.new
+    releases = @client.releases("google/sentencepiece")
+    current_release = releases.first
+    @tag_name = current_release.tag_name
```
Comment (kou) [inline sentencepiece/Rakefile]: Duplicated in `Archive`.

File: sentencepiece/apt/build.sh [hunk 6]
```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```
Comment (kou) [inline sentencepiece/apt/build.sh]: Did you copy this file from the Apache Arrow repository?
Then we should use Apache License 2.0 for this.

**Pull request merged**